src/components/React/Howto.js

```diff
@@ -29,117 +29,133 @@ export default function Howto(props) {
           <h2 id="user-guide" className="mb-4">
```

Comment:
can you try adding `className="px-4"` to the `ModalBody` on line 28?  Right now the text goes right up to the edges of the container and it would be nice to give it a little breathing room

Reply:
Done. I dont think it changed too much. Please check it

Reply:
I don't see the change - can you double check you pushed?